### PR TITLE
GC: area_poly_sphere

### LINKED ISSUE #131: [GA-1594] Great Circle - area_poly_sphere
`area_poly_sphere` NCL function as a part of Great Circle functionality

- [ncl_entry for area_poly_sphere](https://www.ncl.ucar.edu/Document/Functions/Built-in/area_poly_sphere.shtml)
- NCL receipt to area_poly_sphere

### PR DESCRIPTION
**Pull request by cyschneck**
## PR Summary
Covers `area_poly_sphere` on a unit sphere as part of the Great Circle functionality
Covers:
- [area_poly_sphere](https://www.ncl.ucar.edu/Document/Functions/Built-in/area_poly_sphere.shtml)

## Related Tickets & Documents
Closes #131 

## PR Checklist
**General**
- [x] PR includes a summary of changes
- [x] Link relevant issues, make one if none exist
- If adding a new page, select which type:
  - [x] NCL Entry
  - [x] Receipt
- [x] Clear Outputs

### CODE REVIEW

File: ncl/ncl_entries/ncl_entries.rst [hunk 1]
```diff
@@ -38,3 +38,4 @@ and links to additional resources and references.
 
                 - `Climatology <climatology.ipynb>`_
                 - `Meteorology <meteorology.ipynb>`_
+                - `Great Circle <great_circle.ipynb>`_
```
Comment (kafitzgerald) [inline ncl/ncl_entries/ncl_entries.rst:41]:
```suggestion
                - `Great circle <great_circle.ipynb>`_
```

For consistency

File: ncl/ncl_entries/great_circle.ipynb [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Python Resources\n",
+    "- [pyroj.geod() great circle computations](https://pyproj4.github.io/pyproj/stable/api/geod.html)\n",
+    "\n",
+    "## Additional Reading\n",
+    "- [Aviation Formulary for working with Great Circles](https://www.edwilliams.org/avform147.htm)"
```
Comment (kafitzgerald) [inline ncl/ncl_entries/great_circle.ipynb:76]:
```suggestion
    "- [Aviation Formulary for working with great circles](https://www.edwilliams.org/avform147.htm)"
```

File: references.bib [hunk 3]
```diff
@@ -124,3 +124,11 @@ @book{dutton_1986
 	pages={273-274},
 	publisher = {Dover Publications}
 }
+
+@misc{usgs_1987,
+	author = {Snyder, John P.},
+	year = {1987},
+	organization = {U.S Geological Survey},
```
Comment (kafitzgerald) [inline references.bib:131]:
```suggestion
	organization = {U.S. Geological Survey},
```

**Review by kafitzgerald — APPROVED** [review]
Looks great!

Just a few super minor typos

File: ncl/receipts/great_circle.ipynb [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+    "ncl_results[poly_name[6]] = -127516000\n",
+    "ncl_results[poly_name[7]] = 9401.705"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "35803c5d-bf83-4a35-b61c-50466d9d5095",
+   "metadata": {},
+   "outputs": [
```
[the remark this answers is not in the document]
Comment (cyschneck) [reply ncl/receipts/great_circle.ipynb]: Cleared the output, seems to be updated now

**Review by kafitzgerald — APPROVED** [review]

**Review by anissa111 — APPROVED** [review]
I have some concerns, but I believe they are largely due to insufficient infrastructure to support more robust ncl testing and not things that should be addressed before merging this PR.

Sorry again for sitting on this

**Pull request merged**